src/AbstractTween.js

```diff
@@ -45,6 +45,11 @@ function AbstractTween(object, properties) {
 		}
 	}
 
+	// Determine if we are are tweening a CSS object
+	if (typeof CSSStyleDeclaration !== 'undefined') {

```

Comment:
The purpose of the check is actually to prevent a crash when running Tina outside the browser, in node (i.e. benchmarking). I can't actually imagine a case where CSSStyleDeclaration would be undefined (and required) in a typical use case. I have to admit, I haven't written a lot of code where the same code path has to run both in the front end and backend - so if there is a more appropriate pattern to avoid these issues, I'd be happy to hear it. :)

Either way, I will make that clearer in the comment.